test/unofficial-vectors.config

```diff
@@ -0,0 +1,37 @@
+%% -*- mode: erlang; erlang-indent-level: 4; indent-tabs-mode: nil -*-
```

Comment:
I'm assuming this file is left untouched except for the occasional static analysis -related fix. Is my assumption correct?

Reply:
I'm assuming static analysis doesn't apply here, given that this is test data.